web/docs/database/backends.md

```diff
@@ -1,3 +1,182 @@
+
+## Database configuration
+
+You can tall Wasp which database to use in the `app` declaration's `db` field:
+
+```wasp
+app MyApp {
+  title: "My app",
+  // ...
+  db: {
+    system: PostgreSQL,
+    seeds: [
+      import devSeed from "@server/dbSeeds.js"
+    ],
+    prisma: {
+      clientPreviewFeatures: ["extendedWhereUnique"]
+    }
+  }
+}
+```
+
+`app.db` is a dictionary with following fields (all fields are optional):
+
+- `system: DbSystem`
+
+  The database system Wasp will use. It can be either `PostgreSQL` or `SQLite`.
+  If not defined, or even if whole `db` field is not present, default value is `SQLite`.
+  If you add/remove/modify `db` field, run `wasp db migrate-dev` to apply the changes.
+
+- `seeds: [ServerImport]`
+
+  Defines seed functions that you can use via `wasp db seed` to seed your database with initial data.
+  Check out [Seeding](#seeding) section for more details.
+
+- `prisma: [PrismaOptions]`
+
+  Additional configuration for Prisma.
+  It currently only supports a single field:
+
+- `clientPreviewFeatures : string`
+
+  Allows you to define [Prisma client preview features](https://www.prisma.io/docs/concepts/components/preview-features/client-preview-features).
+
+### SQLite
+
+Default database is `SQLite`, since it is great for getting started with a new project (needs no configuring), but it can be used only in development - once you want to deploy Wasp to production you will need to switch to `PostgreSQL` and stick with it.
+Check below for more details on how to migrate from SQLite to PostgreSQL.
+
+### PostgreSQL
+
+When using `PostgreSQL` as your database (`app: { db: { system: PostgreSQL } }`), you will need to make sure you have a postgres database running during development (when running `wasp start` or doing `wasp db ...` commands).
+
+### Using Wasp provided dev database
+
+Wasp provides `wasp start db` command that starts the default dev db for you.
+
+Your Wasp app will automatically connect to it once you have it running via `wasp start db`, no additional configuration is needed. This command relies on Docker being installed on your machine.
+
+### Connecting to an existing database
+
+If instead of using `wasp start db` you would rather spin up your own dev database or connect to some external database, you will need to provide Wasp with `DATABASE_URL` environment variable that Wasp will use to connect to it.
+
+The easiest way to provide the needed `DATABASE_URL` environment variable is by adding it to the [.env.server](https://wasp-lang.dev/docs/language/features#env) file in the root dir of your Wasp project (if that file doesn't yet exist, create it).
+
+You can also set it per command by doing `DATABASE_URL=<my-db-url> wasp ...` -> this can be useful if you want to run specific `wasp` command on a specific database.
```

Comment:
I would avoid using `->` since it makes the docs feel like a draft. Let's use proper interpunction 😄